src/components/channel-forms.js

```diff
@@ -127,18 +121,19 @@ export function DeleteForm({channel, onSubmit}) {
 		try {
 			res = await onSubmit(channel)
 			if (res && res.error) throw res.error
+			console.log('deleted channel')
+			window.location.reload()
```

Comment:
i know this is not the way to do it but it does make the flow a lot easier until we find a better solution